kwave/executor.py

```diff
@@ -0,0 +1,46 @@
+import stat
+import sys
+from pathlib import Path
+import os
+import h5py
+import numpy as np
+
+
+class Executor:
+
+    def __init__(self, use_gpu_if_possible):
```

Comment:
Explicit device selecting might be better and more...

perhaps if GPU avail and running in CPU print info telling USER acceleration is possible with GPU...

OR flag for Fastest device execution which disregards explicit device selection